var/spack/repos/builtin/packages/ffmpeg/package.py

```diff
@@ -173,13 +175,14 @@ def configure_args(self):
             "libvorbis",
             "libvpx",
             "libx264",
-            "avresample",
             "nonfree",
             "openssl",
             "shared",
             "version3",
         ]
 
+        if spec.satisfies("@:4.4"):
+            variant_opts.append("avresample")
```

Comment:
Spack supports conditional variants (https://spack.readthedocs.io/en/latest/packaging_guide.html#conditional-variants) that are likely more readable (IMHO) and can be used with the `enable_or_disable` helper (https://spack.readthedocs.io/en/latest/build_systems/autotoolspackage.html#enable-or-disable).

Reply:
I agree, but I wasn't feeling up to the task today of reworking the variant logic on this when it was otherwise working. I do think it needs some attention though.

I also suspect the libaom and xml2 conflicts aren't relevant any more.